### PR TITLE
updates two_sum function and tests

### PR DESCRIPTION
**Pull request by staceyjo**

### CODE REVIEW

File: functions/two_sum.py [hunk 1]
```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
+        for j in range(i + 1, len(n)):
+            if n[j] == t - n[i]:
+                return [i, j]
```
Comment (scrambledmegs) [inline functions/two_sum.py:5]: Little concerning with time complexity :)

File: functions/two_sum.py [hunk 2]
```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
+    for i in range(len(n)):
+        for j in range(i + 1, len(n)):
```
Comment (amandajones1996) [inline functions/two_sum.py:3]: Great work! Think about the time complexity of your solution. Right now its O(n2). Here a good implementation. 
 ' ' ' python
for a in range:
'''

File: tests/two_sum_test.py [hunk 3]
```diff
@@ -1,2 +1,37 @@
...
+def test_returns_0_1_for_2_7_11_15_with_target_9():
+    # Arrange
+    n = [2, 7, 11, 15]
+    t = 9
+
+    # Act
+    answer = twoSum(n, t)
+
+    # Assert
+    assert answer == [0, 1]
```
Comment (argoodm) [inline tests/two_sum_test.py:13]: Great code! Very clear cut tests. I wonder what other nominal cases we can think of.

**Review by argoodm — COMMENTED** [review]
Love it!

**Review by amandajones1996 — COMMENTED** [review]
See comments:

**Review by scrambledmegs — APPROVED** [review]
Great job! :)

File: functions/two_sum.py (hunk 1 above)
Comment (staceyjo) [reply functions/two_sum.py:5]: ### no

File: functions/two_sum.py (hunk 2 above)
Comment (staceyjo) [reply functions/two_sum.py:3]: ### **Nope**

**Pull request merged**